frontend/pages/investor/[id]/index.tsx

```diff
@@ -15,104 +18,128 @@ import LayoutContainer from 'components/layout-container';
-import { Enum } from 'types/enums';
+import { GroupedEnums } from 'types/enums';
+import { Investor } from 'types/investor';
+
+import { getEnums } from 'services/enums/enumService';
+import { getInvestor } from 'services/investors/investorsService';
+
+export const getServerSideProps = async ({ params: { id }, locale }) => {
+  let investor = null;
+
+  // If getting the project fails, it's most likely because the record has not been found. Let's return a 404. Anything else will trigger a 500 by default.
+  try {
+    ({ data: investor } = await getInvestor(id));
+  } catch (e) {
+    return { notFound: true };
+  }
+
+  const enums = await getEnums();
 
-export async function getServerSideProps(ctx) {
   return {
     props: {
-      intlMessages: await loadI18nMessages(ctx),
+      intlMessages: await loadI18nMessages({ locale }),
+      enums: groupBy(enums, 'type'),
+      investor: decycle(investor),
     },
   };
-}
-
-const InvestorPage: PageComponent<{}, StaticPageLayoutProps> = (props) => {
-  const intl = useIntl();
-  // const { query } = useRouter();
-  // const { id } = query;
-
-  const aboutInfo: {
-    logo: string;
-    name: string;
-    description: string;
-    text: string;
-    website: string;
-    social: SocialType[];
-    contact: ContactItemType;
-  } = {
-    name: 'NESst',
-    description: 'Non-VC Investment vehicle',
-    text: 'NESsT finances enterprises that generate dignified employment and sustain the planet. Since 1997, we have financed 200 enterprises that have sustained jobs for 70,000 individuals and improved 670,000 lives. We achieve our mission through an Incubator that provides pre-seed capital and business services to improve investment readiness; and the Enterprise Fund that deploys seed-state loans to enterprises seeking capital to scale',
-    logo: '/images/temp-placeholders/nesst-logo.png',
-    website: 'https://www.site.com',
-    social: [
-      { id: 'linked-in', url: 'https://www.linkedin.com' },
-      { id: 'twitter', url: 'https://www.twitter.com' },
-      { id: 'facebook', url: 'https://www.facebook.com' },
-      { id: 'instagram', url: 'https://www.instagram.com' },
-    ],
-    contact: {
-      name: 'User name',
-      email: 'user@example.com',
-      phone: '251912343242342',
-    },
+};
+
+type InvestorPageProps = {
+  investor: Investor;
+  enums: GroupedEnums;
+};
+
+const InvestorPage: PageComponent<InvestorPageProps, StaticPageLayoutProps> = ({
+  investor,
+  enums,
+}) => {
+  const {
+    name,
+    website,
+    twitter,
+    facebook,
+    linkedin,
+    instagram,
+    contact_email,
+    contact_phone,
+    categories,
+    picture,
+    ticket_sizes,
+    instrument_types,
+    impacts,
+    sdgs,
+    about,
+    mission,
+    investor_type,
+    other_information,
+    language,
+    prioritized_projects_description,
+  } = investor;
+  const { category, ticket_size, instrument_type, impact, investor_type: investor_types } = enums;
```

Comment:
Nothing wrong with the current implementation, but I'd replace this with: 

```
const { category: allCategories, ticket_size: allTicketSizes, instrument_type: allInstrumentTypes, impact: allImpacts, investor_type: allInvestorTypes } = enums;
````

Just to improve code readability. I found it confusing to read the rest of the [code below](https://github.com/Vizzuality/heco-invest/pull/211/files#diff-473df3881d2d05ea0ec5a8d66693c9202b3c48820f305246b4f80d8eec3d5d73R103) as at a glance it was not very clear which array  was coming from the `enums` and which one from the `investor` endpoint.